packages/web/src/js/utils/ComponentFunctions.ts

```diff
@@ -26,8 +26,19 @@ const onClickHandler = (
   });
 };
 
-const onLoadHandler = (element: HTMLElement, component: typeof BaseComponent) => {
-  component.getOrCreateInstance(element);
+const onLoadHandler = (
+  element: HTMLElement,
+  component: typeof BaseComponent,
+  method: string,
+  event: Event,
+  aim: Aim = 'trigger',
+) => {
+  if (aim === 'target') {
```

Comment:
In this condition, only the correct variable should be set. There is no need to repeat `component.getOrCreateInstance(<variable>)`. Only set the right variably based on the codition.

Reply:
condition changed